docs/txn-use.md

```diff
@@ -1,148 +1,166 @@
-String subName = "shared-subscription";
-String producerName = "txn-message-producer";
-try {
-  @Cleanup
-  Producer<String> sinkProducer = pulsarClient.newProducer(Schema.STRING)
-          .topic(sinkTopicName)
-          .producerName(producerName)
-          .create();
-  @Cleanup
-  Consumer<String> sourceConsumer = pulsarClient.newConsumer(Schema.STRING)
-          .topic(sourceTopicName)
-          .subscriptionName(subName)
-          .subscriptionType(SubscriptionType.Shared)
-          .subscribe();
-  Message<String> message = null;
-  Transaction transaction = null;
-  while (true) {
-      try {
-          message = sourceConsumer.receive();
-          //Open a transaction to handle the received message
-          transaction = pulsarClient.newTransaction()
-                  .withTransactionTimeout(5, TimeUnit.SECONDS)
-                  .build()
-                  .get();
-          //Do some things there
-          //Send message to another topic
-          sinkProducer.newMessage(transaction)
-                  .value("handle message " + message.getValue())
-                  .send();
-          //Ack the message that has been consumed
-          sourceConsumer.acknowledgeAsync(message.getMessageId(), transaction).get();
-          //Commit the transaction
-          transaction.commit().get();
-      } catch (ExecutionException e) {
-          Throwable exception = e.getCause();
-          if (!(exception instanceof PulsarClientException.TransactionConflictException)) {
-              //The message may not be handled, so we need to redeliver it
-              sourceConsumer.negativeAcknowledge(message);
-          }
-          if (!(exception instanceof TransactionCoordinatorClientException.TransactionNotFoundException) && transaction !=null) {
-              //Abort the transaction if there is an exception and the transaction is not end.
-              transaction.abort().get();
-          }
-      }
-  }
-} catch (Exception e) {
-  log.error("Catch Exception", e);
-}
-```
+    **By default**, Pulsar transactions are **disabled**.
+
+    :::
+
+2. Initialize transaction coordinator metadata
+
+    The transaction coordinator can leverage the advantages of partitioned topics (such as load balance).
+
+    **Input**
+
+    ```shell
+    bin/pulsar initialize-transaction-coordinator-metadata -cs 127.0.0.1:2181 -c standalone
+    ```
+
+    **Output**
+
+    ```shell
+    Transaction coordinator metadata setup success
+    ```
+
+3. Create a Pulsar client and enable transactions.
+   
+4. Create producers and consumers.
+
+5. Produce and receive messages.
+
+6. Create transactions.
+
+7. Produce and ack messages with transactions.
+
+    :::note
+
+    Currently, messages can be acked individually rather than cumulatively.
+
+    :::
+
+8. End transactions.
+
+    :::tip
+
+    The code snippet below is the example for step 3 - step 8.
+
+    :::
+
+    **Input**
+
+    ```java
```

Comment:
line 78 - 157: this chunk of code can not be split into several smaller blocks since they are dependent on each other, so I add the tip (line 70) above.